runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -545,6 +537,41 @@ mod development {
 			.into_account_truncating()
 		}
 
+		/// Invests the provided amount via `pallet_investments` to not block
+		/// foreign investments for a different foreign currency.
+		pub fn do_initial_invest_short_cut<T: Runtime + FudgeSupport>(
+			pool_id: u64,
+			amount: Balance,
+			investor: AccountId,
+		) {
+			let pool_currency: CurrencyId = pallet_pool_system::Pallet::<T>::currency_for(pool_id)
+				.expect("Pool existence checked already");
+
+			// Make investor the MembersListAdmin of this Pool
+			assert_ok!(pallet_permissions::Pallet::<T>::add(
```

Comment:
NIT. There is a `give_pool_role()` utility in `generic/utils/mod.rs` just for this

Reply:
[89eaa3f](https://github.com/centrifuge/centrifuge-chain/pull/1700/commits/89eaa3f474af7f356ef5c0e168ef1694ab07de27)